modules/gitops-argo-cd-properties.adoc

```diff
@@ -11,19 +11,28 @@ The Argo CD Custom Resource consists of the following properties:
 [options="header"]
 |===
 |Name |Description |Default |Properties
+
 |`applicationInstanceLabelKey` |The `metadata.label` key name where Argo CD injects the app name as a tracking label.|`app.kubernetes.io/instance` |
 
-|`applicationSet`
-|`applicationSet` controller configuration options.
-| `_<object>_`
-a|* `<image>` - The container image for the `applicationSet` controller. This overrides the `ARGOCD_APPLICATIONSET_IMAGE` environment variable.
+|`aggregatedClusterRoles`    |The aggregation of several cluster roles for a cluster-scoped instance. | `__<empty>__` |
+
+|`applicationSet` |The ApplicationSet Controller configuration options. | `_<object>_`
+a|* `<image>` - The container image for the ApplicationSet Controller. This overrides the `ARGOCD_APPLICATIONSET_IMAGE` environment variable.
   * `<version>` - The tag to use with the `applicationSet` container image.
+  * `<enabled>` - The flag to use to enable the ApplicationSet Controller during Argo CD installation.
+  * `<env>` - Specify the environment for ApplicationSet Controller pods.
```

Comment:
```suggestion
  * `<env>` - Specify the environment for ApplicationSet controller pods.
```

Reply:
Discussed it further with Laura on Slack so I am not fixing this comment to maintain consistency for the word "Controller" throughout the GitOps documentation.